packages/block-editor/src/components/roving-tab-index/README.md

```diff
@@ -0,0 +1,47 @@
+RovingTabIndex
+==============
+
+`RovingTabIndex` exports two separate components `<RovingTabIndex.Container />` and `<RovingTabIndex.Item />` that help to implement a roving tab index.
+
+A roving tab index is helpful for a UI component that contains multiple focusable elements, it reduces the number of tab stops in such an element to a single tab stop. Some more information is available here:
+
+- [WAI Aria Authoring Practices - Roving Tab Index](https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_roving_tabindex)
+
+Some patterns that implement a roving tab index are:
+
+- [Layout Grid](https://www.w3.org/TR/wai-aria-practices/examples/grid/LayoutGrids.html)
+- [Editor Menu Bar](https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-2/menubar-2.html)
```

Comment:
Noting that for the dropdown menus we use [NavigableMenu](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/navigable-container/menu.js) from `@wordpress/components` which is based on more general [NavigableContainer](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/navigable-container/container.js) that also implements roving but limited to horizontal or vertical orientation.

Reply:
I couldn't see that they do implement the roving tab index in the traditional sense. For example, in the editor more menu it's still possible to tab through the individual menu items. If it were a roving tab index the entire menu would only a single tab stop.

It should be possible to use this component to make it behave with a roving tab index, though. If we wanted to 😄.

Reply:
The current tabbing behavior for the dropdown menu is broken, similarly to the legacy version of toolbars :)

It should be all refactored to use roving. This is why I raised it and marked as something to coordinate for the future so we don't maintain 3 different ways to deal with roving.

Reply:
I'm not sure we'd want to have roving behaviour on the more menu. It's a one dimensional dropdown that has to be clicked on to open, so I would expect to be able to tab through it.

Reply:
@tellthemachines, to be clear. The more menu supports both arrows down/up and tabbing for navigation between menu items as of today. You can also use the arrow down key to open this dropdown.